script/run/run.sh

```diff
@@ -21,10 +21,10 @@ IMAGE="docker/compose:$VERSION"
 
 # Setup options for connecting to docker host
 if [ -z "$DOCKER_HOST" ]; then
-    DOCKER_HOST="/var/run/docker.sock"
+    DOCKER_HOST="unix:///var/run/docker.sock"
```

Comment:
Nothing is expanded here, should be single quotes

DOCKER_HOST='unix:///var/run/docker.sock'

(or, actually, no quotes at all since they are not needed here but preferred by many anyway to clearly indicate that there is a string)